packages/compat/src/v1-app.ts

```diff
@@ -205,42 +205,40 @@ export default class V1App implements V1Package {
 
   get htmlTree() {
     if (this.app.tests) {
-      return mergeTrees([this.indexTree, this.app.testIndex()]);
-    }
-    {
+      return mergeTrees([this.indexTree, this.testIndexTree]);
+    } else {
       return this.indexTree;
     }
   }
 
   get indexTree() {
     let indexFilePath = this.app.options.outputPaths.app.html;
-
-    let index: Tree = new Funnel(this.app.trees.app, {
+    let index = new Funnel(this.app.trees.app, {
       allowEmpty: true,
       include: [`index.html`],
       getDestinationPath: () => indexFilePath,
       annotation: 'app/index.html',
     });
-
-    if (this.isModuleUnification) {
```

Comment:
This wasn't directly related to this PR but it's dead code now, and it was in the area.